package.json

```diff
@@ -73,6 +74,15 @@
     "testPathDirs": [
       "./source"
     ],
+    "coverageReporters": ["lcov"],
+    "collectCoverageFrom": [
+      "source/**/*.js",
+      "!source/vendor/**",
+      "!source/demo/**",
+      "!source/jest-setup.js",
+      "!source/TestUtils.js",
+      "!**/*.example.js"
```

Comment:
This should be configured to include all js files that are not in the demo, vendor, or .example.js

Reply:
I would say the main thing to decide here would be whether or not we should include the vendor dir in the coverage report